src/nvapi_interface.cpp

```diff
@@ -8,7 +8,7 @@ extern "C" {
 
     static std::unordered_map<NvU32, void*> registry;
 
-    void* nvapi_QueryInterface(NvU32 id) {
+    __declspec(dllexport) void* __cdecl nvapi_QueryInterface(NvU32 id) {
```

Comment:
Is `__cdecl` actually needed here? I believe it should be the default calling convention anyway.

Reply:
We have __cdecl for other NvAPI declarations, so this is more like a "why not also here" thing.